code/modules/shuttles/escape_pods.dm

```diff
@@ -16,14 +16,14 @@
 /datum/shuttle/ferry/escape_pod/can_cancel()
 	return 0
 
-	
+
 //This controller goes on the escape pod itself
 /obj/machinery/embedded_controller/radio/simple_docking_controller/escape_pod
 	name = "escape pod controller"
 	var/datum/shuttle/ferry/escape_pod/pod
 
-/obj/machinery/embedded_controller/radio/simple_docking_controller/escape_pod/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
-	var/data[0]
+/obj/machinery/embedded_controller/radio/simple_docking_controller/escape_pod/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1);
+/*	var/data[0]
```

Comment:
i intend to eventually rip this entire system out because fuck embedded controllers